tests/framework/CodeInterfaceTests/CobraTF/test1_cobratf_code_interface.xml

```diff
@@ -62,7 +62,7 @@
   </Samplers>
 
   <Steps>
-    <MultiRun name="testDummyStep_MC">
+    <MultiRun name="testDummyStep_MC" clearRunDir="False">
```

Comment:
See the comment about the default. All the tests modifications wouldn't be needed if we keep the user "accountable" of the choice to delete files. 

@joshua-cogliati-inl @wangcj05 @mandd @Jimmy-INL  what do you think?

Reply:
It's true we could avoid modifying test cases, but I think protecting the user from RAVEN reading CSVs when it shouldn't is more beneficial than avoiding the clearRunDir in interfaceCheck inputs.

Reply:
rethinking about this, I agree. 
Good as it is

Reply:
I also would like to automatically set this flag to False when an interfaceCheck is activated... but no need to change it...I might do it tomorrow and create a new PR.